inst/Upgrades.qml

```diff
@@ -651,9 +651,9 @@ Upgrades
 
 		ChangeJS
 		{
-			name:		"inferenceOutputScale"
```

Comment:
Why did you move the name property? I think changeJS should have the name of the option you want to change.

Reply:
I also did not change this however, it was added in the rebase. Not sure how that could have happened? I will ask Don tomorrow, maybe he understands.